docs/website/docs/general-usage/resource.md

```diff
@@ -223,28 +223,10 @@ In the example above, `user_details` will receive data from default instance of
 pipeline.run(users(limit=100) | user_details)
 ```
 
```

Comment:
```suggestion

:::tip
Transformers are allowed not only to **yield** but also to **return** values and can decorate **async** functions and [**async generators**](../reference/performance.md#extract). Below we decorate an async function and request details on two pokemons. Http calls are made in parallel via httpx library.
```python
import dlt
import httpx


@dlt.transformer
async def pokemon(id):
    async with httpx.AsyncClient() as client:
        r = await client.get(f"https://pokeapi.co/api/v2/pokemon/{id}")
        return r.json()

# get bulbasaur and ivysaur (you need dlt 0.4.6 for pipe operator working with lists)
print(list([1,2] | pokemon()))
```
:::
```